modules/mobile/spec/request/decision_letters_request_spec.rb

```diff
@@ -12,6 +12,11 @@
     allow(VBMS::Client).to receive(:from_env_vars).and_return(FakeVBMS.new)
     allow_any_instance_of(IAMUser).to receive(:icn).and_return('24811694708759028')
     iam_sign_in(build(:iam_user))
+    Flipper.enable('mobile_filter_doc_27_decision_letters_out')
+  end
+
+  after do
+    Flipper.disable('mobile_filter_doc_27_decision_letters_out')
```

Comment:
I don't think`enable_in_development: false` disables for specs but I could be wrong. I think it will just be automatically enabled for all specs. Worth checking if you haven't.

Reply:
I've been having to do specs only through CI test runs since this spec is still broken locally. I tried to fix it again but didn't get anywhere. So just wanted to minimize the amount of changes. It should probably be added though. Added.

Reply:
I tested this and it turns out that `enable_in_development: false` does set it to false in specs. This all looks good.